### PR TITLE
feat: sync job info with online system table

### LINKED ISSUE #723: Use system table to store job info
Need to get and read system table of `jobs` for TaskManager.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Re-define the JobInfoManager to create and manage job info
* Sync job info with system table
* Provide method to create system table
* Resolve https://github.com/4paradigm/OpenMLDB/issues/723

### CODE REVIEW

File: java/openmldb-taskmanager/pom.xml [hunk 1]
```diff
@@ -140,6 +140,13 @@
             <version>13.0.0</version>
         </dependency>
 
+        <!-- OpenMLDB client -->
+        <dependency>
+            <groupId>com.4paradigm.openmldb</groupId>
+            <artifactId>openmldb-jdbc</artifactId>
+            <version>0.3.1-allinone</version>
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/pom.xml:147]: remove allinone

File: java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobInfo.java [hunk 2]
```diff
@@ -13,16 +13,33 @@
 
 public class JobInfo {
 
+    public static String[] FINAL_State = new String[] {"finished", "failed", "killed", "lost"};
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobInfo.java]: FINAL_STATE

File: java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobInfo.java [hunk 3]
```diff
@@ -13,16 +13,33 @@
...
+
+    public JobInfo(int id, String jobType, String state, Timestamp startTime, Timestamp endTime, String parameter,
+                   String applicationId, String error) {
+        this.id = id;
+        this.jobType = jobType;
+        this.state = state;
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.parameter = parameter;
+        this.applicationId = applicationId;
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobInfo.java:37]: Add cluster to identity k8s or yarn.

File: java/openmldb-taskmanager/pom.xml (hunk 1 above)
Comment (tobegit3hub) [reply java/openmldb-taskmanager/pom.xml:147]: Should use allinone or versions which depends on profiles.

**Review by aceforeverd — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[Implementation of TaskManager](https://trello.com/c/EkxZMO7i/28-implementation-of-taskmanager)

**Pull request merged**